### ISSUE TITLE
[Quest] Flight of the Wintergarde Defender Dosent work

### ISSUE THREAD

**Issue by anzo31** [WoWManiaUK/Redemption#1676]
In this quest there was some village that should spawn
"Flight of the Wintergarde Defender" http://www.wow-mania.com/armory/?quest=12237.

But now I had looked for 5 min in where they should be and there is none: It might be buged or dosent work.

Edited: " I saw 1 just now but it only wear there for like 1 sec again not sure if its right or not.

**Comment by fvicaria** [comment]
Please search before creating a new issue.
This is a phase issue. There is some pre-quest that you missed.

**Comment by anzo31** [comment]
I did fine then keep that quest broken I just try helping and bump into this one while I lv sorry guys this quest is broken and not fixed

**Comment by fvicaria** [comment]
I will look into it. But not because of your little tantrum but because somebody else bothered to explain the situation better.

**Comment by anzo31** [comment]
So why is it you get so upset with me when I post about a quest how else do you want me to act. Im g lad someone else post because I like the server. But looks like the team dont like when player post about bugs so im sorry for doing so next time I just leave and ignore the bugs I fine

**Comment by fvicaria** [comment]
Info needed:
1. What (quest info, objectives, link)
2. What happens
3. What is expected to happen

Also be prepared to be wrong. We have done these millions of times and sometimes there are tricks and details that the player is not aware of.

We (mainly I) may close an issue if we cannot reproduce it. It doesn't mean that we are always right. For many reasons, it may work for one player and not for another. Sometimes it's difficult to spot it.

**Comment by anzo31** [comment]
Okay then Im sorry for the bad way to explain the bug. I know this is hard im a programmer my self and now how much code and there go into some of this. I always before I post here go on YouTube to see how the quest look on blizzard live servers. All I could see with this quest was they villagers you shall help dont spawn as much as they do on the live servers Like I edited I did saw on spawn but it despawn very fast again. Is that a better way to say it?

**Comment by fvicaria** [comment]
Yes in Redemption the Villager stays alive screaming for about 2 seconds and then dies for no apparent reason. This does not happen in dev. Really frustrating.

**Comment by anzo31** [comment]
I see it a problem that not easy fixed or?

**Comment by fvicaria** [comment]
It's a problem that is annoying the hell out of me since it should not exist. :)
It works perfectly on the PTR with same code and DB.

**Comment by anzo31** [comment]
Yah that sure sounds wired. I understand that I self code a lot but i dunno about the code use in that tho only that I basic know about coding that that not help much but it sure sound wired.

**Comment by anzo31** [comment]
Now they spawn more often but they still despawn very fast

**Comment by fvicaria** [comment]
Fix is still not in Red.
